projects/smoke_adt7476a/src/main.c

```diff
@@ -0,0 +1,76 @@
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
+#include "gpio.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
```

Comment:
add a comment here (or anywhere near the top) describing the expected behaviour, and document any values easily configurable (e.g. the interval and the increment), also the I2C address since we might be testing on a board with only one fan controller